Assets/Scripts/Spawning/ResourceSpawning/ResourceSpawner.cs

```diff
@@ -0,0 +1,86 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Examen.Spawning.ResourceSpawning
+{
+    public class ResourceSpawner : MonoBehaviour
+    {
+        [SerializeField] private List<SpawnAreas> spawnAreas;
+
+        /// <summary>
+        /// Spawns randomly resources
+        /// </summary>
+        public void InitializedSpawning()
```

Comment:
Could you look into shortening the method because currently it is almost 50 lines long and has quite a few indentations